shared/js/media/media_frame.js

```diff
@@ -51,8 +51,26 @@ function MediaFrame(container, includeVideo, maxImageSize) {
     this.video = new VideoPlayer(container);
     this.video.hide();
   }
+
+  // Add a class to the container so we could find it later and use it as
+  // a key in the instance weakmap.
+  this.container.classList.add('media-frame');
+  MediaFrame.instancesToLocalize.set(container, this);
```

Comment:
line 57 uses this.container, and line 58 uses container.  They have the same value, but the code is confusing this way. Change one line or the other to be consistent.